azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java

```diff
@@ -79,11 +84,10 @@
 import javax.inject.Named;
 import javax.inject.Singleton;
 import org.apache.commons.lang.exception.ExceptionUtils;
+import org.apache.log4j.Logger;
```

Comment:
Slf4j ?

Reply:
It was slf4j but the logger is passed into classes which use log4j and share common code with ExecutorServer.
Also, my comment above about slf4j in general.